sdk/search/search-documents/samples-dev/bufferedSenderAutoFlushSize.ts

```diff
@@ -14,7 +14,7 @@ import {
 } from "@azure/search-documents";
 import { createIndex, documentKeyRetriever, WAIT_TIME } from "./setup";
 import { Hotel } from "./interfaces";
-import { delay } from "@azure/core-http";
+import { delay } from "../src/serviceUtils";
```

Comment:
would this work for the published standalone samples?

Reply:
Not Sure. remove this and added the delay function in sample utils